lib/basedriver/commands/index.js

```diff
@@ -9,7 +9,7 @@ Object.assign(
   sessionCmds,
   settingsCmds,
   timeoutCmds,
-  findCmds
+  findCmds,
```

Comment:
is this comma needed?

Reply:
No but it doesn't hurt. It's actually pretty common practice and some linters even require a dangling comma for line separated lists.